src/app/Console/Commands/RequireDevTools.php

```diff
@@ -0,0 +1,154 @@
+        if (! $details) {
+            $this->info(' Creating auth.json file with your authentication token');
+
+            $this->line(' (Find your access token details on https://backpackforlaravel.com/user/tokens)');
+            $username = $this->ask('Access token username');
+            $password = $this->ask('Access token password');
+
+            $process = new Process(['composer', 'config', 'http-basic.backpackforlaravel.com', $username, $password]);
+            $process->run(function ($type, $buffer) use ($username, $password) {
+                if ($type === Process::ERR) {
+                    // Fallback
+                    $authFile = [
+                        'http-basic' => [
+                            'backpackforlaravel.com' => [
+                                'username' => $username,
+                                'password' => $password,
+                            ],
+                        ],
+                    ];
+
+                    if (File::exists('auth.json')) {
+                        $currentFile = json_decode(File::get('auth.json'), true);
+                        if (! ($currentFile['http-basic']['backpackforlaravel.com'] ?? false)) {
+                            $authFile = array_merge_recursive($authFile, $currentFile);
+                        }
+                    }
+
+                    File::put('auth.json', json_encode($authFile, JSON_PRETTY_PRINT));
+                }
+                $this->progressBar->advance();
+            });
+        }
+
+        // Check if repositories exists
+        $details = null;
+        $process = new Process(['composer', 'config', 'repositories.backpack/devtools']);
+        $process->run(function ($type, $buffer) use (&$details) {
+            if ($type !== Process::ERR && $buffer !== '') {
+                $details = json_decode($buffer);
+            } else {
+                $details = json_decode(File::get('composer.json'), true)['repositories']['backpack/devtools'] ?? false;
+            }
+            $this->progressBar->advance();
+        });
+
+        // Create repositories
+        if (! $details) {
+            $this->info(' Creating repositories entry in composer.json');
+
+            $process = new Process(['composer', 'config', 'repositories.backpack/devtools', 'vcs', 'https://github.com/laravel-backpack/DevTools']);
+            $process->run(function ($type, $buffer) {
+                if ($type === Process::ERR) {
+                    // Fallback
+                    $composerJson = Str::of(File::get('composer.json'));
+
+                    $currentRepositories = json_decode($composerJson, true)['repositories'] ?? false;
+
+                    $repositories = Str::of(json_encode([
+                        'repositories' => [
+                            'backpack/devtools' => [
+                                'type' => 'vcs',
+                                'url' => 'https://github.com/laravel-backpack/DevTools',
```

Comment:
@promatik , heads-up, the composer repository should NOT point to the Github URL. Only you, Pedro and I can get it from there, because we're the only ones with Github access.

The correct repository is `https://repo.backpackforlaravel.com/` and it's type `composer` not `vcs`